crates/proto/src/version/v671/helper.rs

```diff
@@ -0,0 +1,8 @@
+use crate::helper::ProtoHelper;
+use crate::version::v729::gamepackets::GamePackets;
```

Comment:
are you sure that is the correct import?

Reply:
missed that, should be fixed now for them (v662 was also wrong lol)